features/option-pluck-patch.feature

```diff
@@ -299,3 +299,21 @@ Feature: Option commands have pluck and patch.
       """
       1
       """
+
+  @patch
+  Scenario: When we don't pass all necessary argumants.
+    Given a WP install
+    And an input.json file:
+      """
+      {
+        "foo": "bar"
+      }
+      """
+    And I run `wp option update option_name --format=json < input.json`
+
+    When I try `wp option patch update option_name foo`
+    And STDERR should contain:
+      """
+      Please provide value to update.
+      """
+    And the return code should be 1
```

Comment:
Can we add a test case for:

```
When I run `wp option patch update option_name foo 0`
```

To make sure we're handling falsy values correctly?

Reply:
It should not allow this to update, right?

Reply:
> It should not allow this to update, right?

Perhaps I'm misunderstanding but isn't `0` a valid value?

Reply:
Actually, I misunderstood.

I thought zero should be considered an empty value.

Then I saw other commands in the same `option-pluck-patch.feature` file.

So fixe it.